cpp/src/Ice/ObjectAdapterI.cpp

```diff
@@ -972,13 +972,12 @@ Ice::ObjectAdapterI::initialize(optional<RouterPrx> router)
 
         int threadPoolSize = properties->getPropertyAsInt(_name + ".ThreadPool.Size");
         int threadPoolSizeMax = properties->getPropertyAsInt(_name + ".ThreadPool.SizeMax");
-        bool hasPriority = properties->getProperty(_name + ".ThreadPool.ThreadPriority") != "";
 
         //
         // Create the per-adapter thread pool, if necessary. This is done before the creation of the incoming
         // connection factory as the thread pool is needed during creation for the call to incFdsInUse.
         //
-        if (threadPoolSize > 0 || threadPoolSizeMax > 0 || hasPriority)
```

Comment:
ThreadPriority is not a "key" property in C# or Java, so I removed it from C++ for consistency.